guide/14-deep-learning/add_custom_nlp_models_using_model_extension.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>It is responsible for creating an instance of your NLP class with all the necessary attributes required for processing and analysis. When you create an instance of your NLP class, this method makes sure it has all the required settings and default values. For instance, if your NLP function needs specific settings such as paths to models, or special tokens, this is where you set those up.I think this needs to reworded. Because, we intend to keep <code>__init__</code>lighter. Specifying it to contain set ups may mean something else, like setting up the model&nbsp;</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2007/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>